recipe/meta.yaml

```diff
@@ -70,7 +72,7 @@ requirements:
     - typing-extensions >=4.3.0,<5
```

Comment:
setuptools can be removed from run

Reply:
Done, and while. was there, I added `importlib-metadata` since it was missing.